invenio_communities/models.py

```diff
@@ -320,6 +320,17 @@ def filter_communities(cls, p, so):
             query = query.order_by(db.desc(cls.ranking))
         return query
 
+    @property
+    def oaiset_spec(self):
```

Comment:
Actually that was a leftover, which was supposed to be removed on behalf of the property. But I can remove the property instead.